internal/lm/nvml.go

```diff
@@ -218,6 +231,91 @@ func newGPUModeLabeler(devices []resource.Device) (Labeler, error) {
 	return labels, nil
 }
 
+func newImexDomainLabeler(configFile string, device []resource.Device) (Labeler, error) {
+	if configFile == "" {
+		return nil, nil
+	}
+
+	// Read file and parse it
+	imexConfig, err := os.Open(configFile)
+	if err != nil && os.IsNotExist(err) {
+		return nil, nil
+	}
+	if err != nil {
+		return nil, fmt.Errorf("failed to open imex config file: %v", err)
+	}
```

Comment:
```suggestion
	if os.IsNotExist(err) {
		return nil, nil
	} else if err != nil {
		return nil, fmt.Errorf("failed to open imex config file: %v", err)
	}
```

Reply:
I prefer the first one myself .... I have a hatred for else statements if they can be avoided.